cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -38,6 +38,10 @@ bundle common cfe_internal_hub_vars
         comment => "Root directory of Enterprise Web interface",
         handle => "cfe_internal_hub_vars_docroot";
 
+      "public_docroot" string => ifelse( isdir("$(sys.workdir)/httpd/htdocs/public"), "$(sys.workdir)/httpd/htdocs/public",
+                                               "$(sys.workdir)/httpd/htdocs"),
```

Comment:
Instead of `"$(sys.workdir)/httpd/htdocs"),` how about using the `docroot` variable, `"$(docroot)"),`

Reply:
@nickanderson  done